config/boot.rb

```diff
@@ -1,3 +1,12 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+require 'rails/commands/server'
+module Rails
+  class Server
+    def default_options
+      super.merge(Host:  '0.0.0.0', Port: 3000)
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.